tests/mock_tests/aclorch_ut.cpp

```diff
@@ -628,16 +628,74 @@ namespace aclorch_test
         // consistency validation with CRM
         bool validateResourceCountWithCrm(const AclOrch *aclOrch, CrmOrch *crmOrch)
         {
-            auto resourceMap = Portal::CrmOrchInternal::getResourceMap(crmOrch);
-            auto ifpPortKey = Portal::CrmOrchInternal::getCrmAclKey(crmOrch, SAI_ACL_STAGE_INGRESS, SAI_ACL_BIND_POINT_TYPE_PORT);
-            auto efpPortKey = Portal::CrmOrchInternal::getCrmAclKey(crmOrch, SAI_ACL_STAGE_EGRESS, SAI_ACL_BIND_POINT_TYPE_PORT);
+             // ACL counter
+            {
+                auto const &resourceMap = Portal::CrmOrchInternal::getResourceMap(crmOrch);
+                uint32_t crm_acl_cnt = 0;
+                for (auto const &kv : resourceMap.at(CrmResourceType::CRM_ACL_TABLE).countersMap)
+                {
+                    crm_acl_cnt += kv.second.usedCounter;
+                }
 
-            auto ifpPortAclTableCount = resourceMap.at(CrmResourceType::CRM_ACL_TABLE).countersMap[ifpPortKey].usedCounter;
-            auto efpPortAclTableCount = resourceMap.at(CrmResourceType::CRM_ACL_TABLE).countersMap[efpPortKey].usedCounter;
+                if (crm_acl_cnt != Portal::AclOrchInternal::getAclTables(aclOrch).size())
+                {
+                    ADD_FAILURE() << "ACL table size is not consistent between CrmOrch (" << crm_acl_cnt
+                                  << ") and AclOrch " << Portal::AclOrchInternal::getAclTables(aclOrch).size();
+                    return false;
+                }
+            }
+
+            // Verify ACL Rules
+            //
+            // for each CRM_ACL_ENTRY and CRM_ACL_COUNTER's entry => the ACL TABLE should exist
+            //
+            for (auto acl_entry_or_counter : { CrmResourceType::CRM_ACL_ENTRY, CrmResourceType::CRM_ACL_COUNTER })
+            {
+                auto const &resourceMap = Portal::CrmOrchInternal::getResourceMap(crmOrch);
+                for (auto const &kv : resourceMap.at(acl_entry_or_counter).countersMap)
+                {
+                    auto acl_oid = kv.second.id;
+                    auto acl_id = sai_serialize_object_id(acl_oid);
```

Comment:
I don't think this variable is ever used.

Reply:
yes, will remove it

Reply:
yes, will remove it